test/witness_calc.test.js

```diff
@@ -0,0 +1,117 @@
+import path from "path";
+import {WitnessCalculatorBuilder} from "../main.js";
+import * as fastFile from "fastfile";
+import * as snarkjs from "snarkjs";
+import fs from "fs";
+import {expect} from "chai";
+
+describe("Witness Calc test", function () {
+    const wasmFilename = path.join("test", "circuit", "circuit_js", "circuit.wasm");
+    const zkeyFilename = path.join("test", "circuit", "circuit.zkey");
+    const vkeyFilename = path.join("test", "circuit", "verification_key.json");
+    const input = {
+        "a": 1,
+        "b": 2,
+        "c": 3
+    };
+
+    this.timeout(15000);
+
+    //let curve;
```

Comment:
I would remove comments or add additional explanation why this code is commented out

Reply:
Added comment